php/values.yaml

```diff
@@ -312,10 +312,7 @@ fpm:
 
   # php.ini configuration
   # http://php.net/manual/en/errorfunc.configuration.php
-  ini:
-    displayErrors: false
-    errorReporting: E_ALL
-    exposePhp: false
+  ini: {}
```

Comment:
@cw-niinuma 
What are the use cases for using `ini` and when mounting `php.ini`?